src/typescript-service.ts

```diff
@@ -69,7 +69,7 @@ export class TypeScriptService implements LanguageHandler {
 
 	async initialize(params: InitializeParams, token = CancellationToken.None): Promise<InitializeResult> {
 		if (params.rootUri || params.rootPath) {
-			this.root = params.rootPath || util.uri2path(params.rootUri);
+			this.root = params.rootPath ? util.uri2path(params.rootPath) : util.uri2path(params.rootUri);
```

Comment:
this addition does not break communication with other clients who sent proper data, but allows me to debug TSJS LS server in combination with Sourcegraph w/o need to run other components.
uri2path does not change string unless it looks like uri, so
`uri2path('file:///foo') = '/foo'`
`uri2path('/foo') = '/foo'`